### PR TITLE
Add a CONTRIBUTING.md file

### PR DESCRIPTION
**Pull request by MaudBoucherit**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# Contributing
+
+The `Karl` R package is an open source project,
```
Comment (TedHaley) [inline CONTRIBUTING.md]: Let's change this to say Pythia.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+There are many ways to contribute,
+from writing new pieces of code and improving existing ones
+to updating or filling in the documentation
+and submitting [bug reports][issues]
+about things that don't work, aren't clear, or are missing.
+
+Comments on issues and reviews of pull requests are welcomed:
+we are smarter together than we are on our own.   
+Feedback is appreciated and encouraged as Karl is continually evolving to better serve the community.
```
Comment (TedHaley) [inline CONTRIBUTING.md]: Ditto here

**Comment by MaudBoucherit** [comment]
I fixed it!

**Comment by TedHaley** [comment]
Nice, looks good. Fixed all the Karls to Pythias

**Pull request merged**